cmd.c

```diff
@@ -758,16 +757,32 @@ void cmd_app(char *s)
 	phfs.dataOffs = 0;
 
 	/* Parse command arguments */
-	if (cmd_parseArgs(s, cmdArgs, &cmdArgsc) < 0) {
+	if (cmd_parseArgs(s, cmdArgs, &cmdArgsc) < 0 && (cmdArgsc < 3 || cmdArgsc > 6)) {
```

Comment:
Shouldn't be `|| cmdArgsc < 3 || cmdArgsc > 6`?

Reply:
Yes it should, updated.